.github/workflows/docs.yml

```diff
@@ -24,14 +24,20 @@ jobs:
         with:
           version: 'v0.8.0'
 
-      - name: Compile sa.typ
-        run: typst compile SA/sa.typ
+      - name: Compile documents
+        run: make all
 
-      - name: Upload sa.pdf
+      - name: Upload design_concept.pdf
         uses: actions/upload-artifact@v3.1.3
         with:
-          name: SA
-          path: SA/sa.pdf
+          name: SA Design Concept
+          path: SA/design_concept/design_concept.pdf
+
+      - name: Upload project_documentation.pdf
+        uses: actions/upload-artifact@v3.1.3
+        with:
+          name: SA Project Documentation
```

Comment:
YThis is a technical name, so i would maybe call id `pdfs` or something like that. You'll then need to update it below under `download-artifact` as well

(you should be able to specify a directory as `path`)